internal/commands/pull/inputs.go

```diff
@@ -24,12 +25,18 @@ type inputs struct {
 	LocalPath           string
 	AppVersion          realm.AppConfigVersion
 	IncludeDependencies bool
+	IncludeNodeModules  bool
+	IncludePackageJSON  bool
 	IncludeHosting      bool
 	DryRun              bool
 	TemplateIDs         []string
 }
 
 func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
+	if i.IncludeNodeModules && i.IncludePackageJSON {
```

Comment:
```suggestion
	if (i.IncludeNodeModules || i.IncludeDependencies) && i.IncludePackageJSON {
```

Reply:
I will do the same template error message approach for pull, push and diff. Lemme know if you like them.